src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java

```diff
@@ -102,6 +102,14 @@
     @Parameter(property = "useSnapshotInHotfix", defaultValue = "false")
     private boolean useSnapshotInHotfix;
 
+
+    /**
+     * Whether to merge hotfix into master.
+     * 
+     */
+    @Parameter(property = "skipMergeMaster", defaultValue = "false")
+    private boolean skipMergeMaster = false;
```

Comment:
It is better to name it `skipMergeProduction` or `skipMergeProdBranch` inline with `skipMergeDevBranch`.

Reply:
agreed, I'll change it to  `skipMergeProdBranch`